### ISSUE TITLE
Keras Build A Multi_Layer Beta Quiz throws erroneous error

### ISSUE THREAD

**Issue by Alexspayne** [udacity/sdc-issue-reports#439]
The following code throws an out of range exception.

model.add(Flatten(input_shape=(32,32,3)))
model.add(Dense(128, activation='relu'))
model.add(Dense(5, activation ='softmax'))

I imagine this is because the answer validation is expecting standalone activation layers.

**Comment by awbrown90** [comment]
@Alexspayne that's exactly correct, I made the feedback message more clear, and updated it in the Beta. Now everything is under one concept called Keras Lab. The main goal of this concept is to add the Keras jupyter notebook information to the classroom setting.
